CourseGrab/Extensions/UserDefaults+Extensions.swift

```diff
@@ -19,12 +28,12 @@ extension UserDefaults {
         }
     }
     
-    var areNotificationsEnabled: Bool {
+    var storedDeviceToken: String {
```

Comment:
was very confused how you were just getting `UserDefaults.standard.storedDeviceToken` in AppDelegate.. and then saw this -- this is cool